pages/bare_metal_cloud/dedicated_servers/api-os-installation/guide.en-gb.md

```diff
@@ -82,6 +82,23 @@ You can find interesting information such as the following:
 
 Some OSs can have a list of specific questions. If that is the case, the `inputs` key contains a list of questions.
 
+Example of specific questions for Debian 12 (Bookworm):
+
+```json
+{
+    "inputs": [
+        {
+            "default": "",
+            "name": "sshKey",
+            "mandatory": false,
+            "enum": [],
+            "description": "SSH Public Key",
```

Comment:
Please do not translate this, do not translate this everything within "json" code tags